helm/hpcc/values.schema.json

```diff
@@ -1453,6 +1465,14 @@
         }
       }
     },
+    "minStartupTime": {
+      "description": "The time to wait before initiating startup probing (in seconds). Default 0",
+      "type": "string"
```

Comment:
I think you mean integer...

Reply:
Interesting. It didn't complain! I will fix